src/Toolkit/src/Command/UxToolkitInstallCommand.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Ux\Toolkit\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Ux\Toolkit\ComponentRepository\ComponentIdentifier;
+use Symfony\Ux\Toolkit\ComponentRepository\RepositoryFactory;
+
+/**
+ * @author Jean-François Lépine
+ *
+ * @internal
+ */
+#[AsCommand(
+    name: 'ux:toolkit:install',
+    description: 'This command will install a new UX Component in your project',
+)]
+class UxToolkitInstallCommand extends Command
+{
+    public function __construct(
+        private readonly RepositoryFactory $repositoryFactory,
+        private readonly ComponentIdentifier $componentIdentifier,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('component', InputArgument::REQUIRED, 'The component name (Ex: button) or repository URL')
+            ->addOption(
+                'destination',
+                'd',
+                InputArgument::OPTIONAL,
+                'The destination directory',
+                'templates/components/ux'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $name = $input->getArgument('component');
+
+        try {
+            $component = $this->componentIdentifier->identify($name);
+
+            // Get the correct source (remote or official)
+            $repository = $this->repositoryFactory->factory($component);
+        } catch (\Throwable $e) {
+            $io->error($e->getMessage());
+
+            return Command::FAILURE;
+        }
+
+        if ($io->isVerbose()) {
+            $io->text('Component information:');
+            $io->table(['Vendor', 'Package', 'Version', 'Name'], [
+                [$component->getVendor(), $component->getPackage(), $component->getVersion(), $component->getName()],
+            ]);
+        }
+
+        // Prepare destination
+        $directory = $input->getOption('destination');
+        if (empty($directory)) {
+            $io->error('The destination directory is not valid');
+
+            return Command::FAILURE;
+        }
+
+        if (!file_exists(\dirname($directory)) || !is_writable(\dirname($directory))) {
+            $io->error(\sprintf('The directory "%s" does not exist or is not writable', \dirname($directory)));
```

Comment:
```suggestion
            $io->error(\sprintf('The directory "%s" does not exist or is not writable.', \dirname($directory)));
```

Same for every other one